docs/get-started/store-and-retrieve/store-data.md

```diff
@@ -8,7 +8,7 @@ description: Start storing your data on the Filecoin network. This section cover
 Start storing your data on the Filecoin network. This section covers packaging your data, importing it into your local Lotus lite-node, finding a storage provider through the Filecoin Plus miner registry, creating a storage deal, and then waiting for the deal to complete! 
 
 :::danger
-Do not store personal data on the Filecoin network, even if it's encrypted. Only store public and static data on the Filecoin network. Access control is on the project roadmap.
+Filecoin is optimized for public data and doesn't yet support access controls. If storing private data, ensure you encrypt it before storage to ensure it remains unreadable by anyone without the ability to decrypt it.
```

Comment:
```suggestion
Filecoin is optimized for public data and doesn't yet support access controls. If storing private data, ensure you encrypt it before storage to ensure it remains unreadable by anyone without the ability to decrypt it. If a vulnerability is found in your encryption process at any point in the future, then your data may be compromised.
```

Reply:
It's important to stress that if a user encrypts their data with GPG, and then throws it on Filecoin and then an exploit in GPG is found, then that user's data could potentially be at risk.

Reply:
fair point. added the suggestion